.github/workflows/badges.yml

```diff
@@ -0,0 +1,129 @@
+name: Generate badges
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  create-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Git
+        run: |
+          git config --global user.name 'Groupings Project'
+          git config --global user.email 'actions@noreply.its.hawaii.edu'
+
+      - name: Create the badges branch if it doesn't exist
+        run: |
+          branch_name="badges"
+          if ! git ls-remote --exit-code origin refs/heads/$branch_name; then
+            echo "Branch $branch_name does not exist. Creating it..."
+            git checkout -b $branch_name
+            git rm -rf .github
+            git rm -rf .
+            git commit -m "Remove .github and all other files"
+            git push -u origin $branch_name
+          else
+            echo "Branch $branch_name already exists."
+          fi
+
+  generate-jest-badges-on-badges-branch:
```

Comment:
We can take out the -on-badges-branch ending here. Same goes for generate-jacoco-badges-on-badges-branch.

Reply:
done